string object
---
bool in_scene
string object_id